.github/workflows/review.yml

```diff
@@ -38,6 +41,9 @@ jobs:
           DOCKER_BUILD_ARG_NEXT_PUBLIC_CMS_GRAPHQL_ENDPOINT: ${{ env.NEXT_PUBLIC_CMS_GRAPHQL_ENDPOINT }}
           DOCKER_BUILD_ARG_NEXT_PUBLIC_UNIFIED_SEARCH_GRAPHQL_ENDPOINT: ${{ env.NEXT_PUBLIC_UNIFIED_SEARCH_GRAPHQL_ENDPOINT }}
           DOCKER_BUILD_ARG_NEXT_PUBLIC_NEXT_API_GRAPHQL_ENDPOINT: ${{ env.CUSTOM_ENVIRONMENT_URL }}/api/graphql
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_MATOMO_URL_BASE: "//webanalytics.digiaiiris.com/js/"
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_MATOMO_SITE_ID: 587
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_MATOMO_ENABLED: "false"
```

Comment:
We've used 0/1 flags for boolean variables because other projects have preferred it previously.